nixos/modules/programs/steam.nix

```
@@ -141,6 +143,28 @@ in {
       Load the extest library into Steam, to translate X11 input events to
       uinput events (e.g. for using Steam Input on Wayland)
     '';
+
+    protontricks = mkOption {
+      description = "Protontricks Steam integration";
+      default = {};
+      type = types.submodule {
+        options = {
```

This does not need to be a submodule.

```suggestion
    protontricks = {
```

---

Why? What's the point of having gamescopeSession as a submodule then?

---

I don't see a reason for it to be a submodule either.

cc @nrdxp